src/osd/scrubber/pg_scrubber.cc

```diff
@@ -1129,10 +1131,106 @@ void PgScrubber::run_callbacks()
   }
 }
 
+void PgScrubber::populate_store(inconsistent_objs_t&& all_errors)
+{
+  dout(10) << __func__ << " " << all_errors.size() << " errors" << dendl;
+  if (all_errors.empty()) {
+    return;
+  }
+  if (state_test(PG_STATE_REPAIR)) {
```

Comment:
This is pretty surprising for a method called populate_store (or as I suggested above persist_scrub_results).  Can we do this check in the caller?

Reply:
Will do (originally placed here so that I was able to perform the preceding check ("is empty") in this function).